broker/src/main/java/io/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -66,7 +67,11 @@
 import org.slf4j.Logger;
 
 public final class ZeebePartition extends Actor
-    implements RaftCommitListener, RaftRoleChangeListener, HealthMonitorable, FailureListener {
+    implements RaftCommitListener,
+        RaftRoleChangeListener,
+        HealthMonitorable,
+        FailureListener,
+        DiskSpaceUsageListener {
```

Comment:
So many listeners :laughing:

Reply:
Just waiting for the `ListenerListener` interface now :eyes: